src/components/documentation/tabs/TechTab.tsx

```
@@ -196,13 +223,13 @@ export const TechTab = () => {
                       <div key={index} className="relative group">
-                        <SyntaxHighlighter
-                          language="typescript"
-                          style={vscDarkPlus}
-                          className="rounded-lg"
-                        >
-                          {code}
-                        </SyntaxHighlighter>
+                        <div className="relative rounded-md bg-[#1E1E1E] p-4 overflow-x-auto">
+                          <div 
+                            dangerouslySetInnerHTML={{ 
+                              __html: highlightedCode[code] || `<pre class="text-white">${code}</pre>`
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)